src/Patchwork.jl

```diff
@@ -31,10 +31,10 @@ export Node,
        MaybeKey,
        tohtml
 
-typealias MaybeKey @compat Union{(@compat Void), Symbol}
+const MaybeKey = @compat Union{(@compat Void), Symbol}
 
 # A Patchwork node
-abstract Node
+@compat abstract type Node end
```

Comment:
you need to update the Compat minimum version when you use new features like this